be/src/column/column_visitor.cpp

```diff
@@ -0,0 +1,56 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "column/column_visitor.h"
+
+namespace starrocks {
+
+#define VISIT_IMPL(ClassName) \
+    Status ColumnVisitor::visit(const ClassName& column) { return Status::NotSupported(#ClassName); }
+
+VISIT_IMPL(vectorized::NullableColumn)
+VISIT_IMPL(vectorized::ConstColumn)
+VISIT_IMPL(vectorized::ArrayColumn)
```

Comment:
Do we need add some type macros for easily use like apache arrow?

Reply:
Sorry, I don't know which kind of macros are you mean, could you please give some examples?